templates/master.yaml.tmpl

```diff
@@ -45,8 +47,8 @@ storage:
                     "type": "calico",
                     "log_level": "info",
                     "datastore_type": "kubernetes",
-                    "nodename": "__KUBERNETES_NODE_NAME__",
-                    "mtu": 1500,
+                    "nodename": "__KUBERNETES_NODE_NAME__",          
+                    "mtu": __CNI_MTU__,
```

Comment:
where is this comes from?

Reply:
I saw this in calico docs:

> Note: If using Kubernetes self-hosted manifests, you should modify the veth_mtu value in the Calico ConfigMap instead, and leave "mtu" here set to __CNI_MTU__.

And it is configured here for azure (for aws was already defined)
https://github.com/giantswarm/giantnetes-terraform/blob/enable_configurable_mtu/templates/master.yaml.tmpl#L37

Reply:
ack, thx